docs/reference/HailExpressionLanguage.md

```diff
@@ -83,6 +83,7 @@ Several Hail commands provide the ability to perform a broad array of computatio
      - forall: `arr.forall(v => expr)` -- Returns a boolean which is true if the array is empty, or `expr` evaluates to `true` for **every** element
      - sort: `arr.sort([ascending])` -- Returns a new array with the same elements in ascending order according to their value, which must be numeric or string. For descending order, use `arr.sort(false)`. Missing elements are always placed at the end.
      - sortBy: `arr.sortBy(v => expr[,ascending])` -- Returns a new array with the same elements in ascending order according to the value of `expr`, which must be numeric or string. For descending order, use `arr.sortBy(v => expr, false)`. Elements with missing `expr` values are always placed at the end.
+     - groupBy: `arr.groupBy(v => v.k)` -- Returns a Dict mapping from keys specified by the lambda argument to an Array of the values associated with that key. The keys must be Strings.
```

Comment:
The Java 8 Streams library [adopts the same interface](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collectors.html#groupingBy-java.util.function.Function-java.util.stream.Collector-). I agree, it's the right way to do `Aggregable.groupBy`.